--- cores/epoxy/Arduino.cpp.orig	2023-02-16 13:12:02.079503763 -0500
+++ cores/epoxy/Arduino.cpp	2023-02-16 13:12:08.143455805 -0500
@@ -110,3 +110,17 @@
     uint8_t /*dataPin*/, uint8_t /*clockPin*/, uint8_t /*bitOrder*/) {
   return 0;
 }
+
+#ifdef EXTENDED_PIN_MODE
+// Platforms who want to declare more than 256 pins need to define EXTENDED_PIN_MODE globally
+typedef uint32_t pin_size_t;
+#else
+typedef uint8_t pin_size_t;
+#endif
+
+// Weak reference so that the calling code can provide its own attachInterrupt() and detachInterrupt().
+void attachInterrupt(pin_size_t interruptNumber, void (*userFunc)(void), PinStatus mode) __attribute__((weak));
+void attachInterrupt(pin_size_t interruptNumber, void (*userFunc)(void), PinStatus mode) {}
+
+void detachInterrupt(pin_size_t interruptNumber) __attribute__((weak));
+void detachInterrupt(pin_size_t interruptNumber) {}
