--- cores/epoxy/Arduino.h.orig	2023-02-16 13:25:35.750251497 -0500
+++ cores/epoxy/Arduino.h	2023-02-16 13:25:56.479809969 -0500
@@ -35,16 +35,17 @@
 #endif
 
 // Used by digitalRead() and digitalWrite()
-#define HIGH 0x1
-#define LOW  0x0
+#if defined(EPOXY_CORE_AVR) || defined(EPOXY_CORE_ESP8266)
+  #define HIGH 0x1
+  #define LOW  0x0
+#endif
 
 // Used by tvsc::bus::gpio::set_mode() and other GPIO functions.
 #if defined(EPOXY_CORE_AVR)
   #define INPUT 0x0
   #define OUTPUT 0x1
   #define INPUT_PULLUP 0x2
-#endif
-#if defined(EPOXY_CORE_ESP8266)
+#elsif defined(EPOXY_CORE_ESP8266)
   #define INPUT             0x00
   #define INPUT_PULLUP      0x02
   #define INPUT_PULLDOWN_16 0x04 // PULLDOWN only possible for pin16
@@ -58,6 +59,22 @@
   #define FUNCTION_2        0x28
   #define FUNCTION_3        0x38
   #define FUNCTION_4        0x48
+#else
+typedef enum {
+  LOW     = 0,
+  HIGH    = 1,
+  CHANGE  = 2,
+  FALLING = 3,
+  RISING  = 4,
+} PinStatus;
+
+typedef enum {
+  INPUT            = 0x0,
+  OUTPUT           = 0x1,
+  INPUT_PULLUP     = 0x2,
+  INPUT_PULLDOWN   = 0x3,
+  OUTPUT_OPENDRAIN = 0x4,
+} PinMode;
 #endif
 
 // Various math constants.
@@ -257,12 +274,18 @@
 void shiftOut(uint8_t dataPin, uint8_t clockPin, uint8_t bitOrder, uint8_t val);
 uint8_t shiftIn(uint8_t dataPin, uint8_t clockPin, uint8_t bitOrder);
 
+#if defined(EPOXY_CORE_ESP8266) || defined(EPOXY_CORE_AVR)
 void attachInterrupt(uint8_t interruptNum, void (*userFunc)(void), int mode);
 void detachInterrupt(uint8_t interruptNum);
 #if defined(EPOXY_CORE_ESP8266)
   void attachInterruptArg(uint8_t pin, void (*)(void*), void* arg, int mode);
 #endif
-
+#else
+void attachInterrupt(uint8_t interruptNum, void (*userFunc)(void), PinStatus mode);
+void detachInterrupt(uint8_t interruptNum);
+void attachInterruptArg(uint8_t pin, void (*)(void*), void* arg, PinStatus mode);
+#endif
+  
 /** Provided in the client code's *.ino file. */
 void setup();
 
