--- bazel/grpc_build_system.bzl.orig	2023-01-19 10:24:15.299810335 -0500
+++ bazel/grpc_build_system.bzl	2023-01-19 10:24:22.643848560 -0500
@@ -88,7 +88,7 @@
         "alt_grpc++_base_unsecure_legacy": PRIVATE,
         "alts_frame_protector": PRIVATE,
         "channelz": PRIVATE,
-        "client_channel": PRIVATE,
+        "client_channel": PUBLIC,
         "cli": PRIVATE,
         "debug_location": PRIVATE,
         "endpoint_tests": PRIVATE,
