--- tools/arm/arm-none-eabi/include/_newlib_version.h.orig	2022-08-02 03:23:43.000000000 -0400
+++ tools/arm/arm-none-eabi/include/_newlib_version.h	2023-03-02 15:32:42.473154188 -0500
@@ -3,16 +3,10 @@
 #ifndef _NEWLIB_VERSION_H__
 #define _NEWLIB_VERSION_H__ 1
 
-/* The newlib version in string format. */
-#define _NEWLIB_VERSION "@NEWLIB_VERSION@"
-
-/* The newlib major version number. */
-#define __NEWLIB__ @NEWLIB_MAJOR_VERSION@
-
-/* The newlib minor version number. */
-#define __NEWLIB_MINOR__ @NEWLIB_MINOR_VERSION@
-
-/* The newlib patch level. */
-#define __NEWLIB_PATCHLEVEL__ @NEWLIB_PATCHLEVEL_VERSION@
+#define _NEWLIB_VERSION "2.4.0"
+#define __NEWLIB__ 2
+#define __NEWLIB_MINOR__ 4
+#define __NEWLIB_PATCHLEVEL__ 0
 
 #endif /* !_NEWLIB_VERSION_H__ */
+
